syntax/parse.go

```diff
@@ -938,3 +947,99 @@ func terminatesExprList(tok Token) bool {
 	}
 	return false
 }
+
+// Comment assignment.
+// We build two lists of all subexpressions, preorder and postorder.
```

Comment:
The term "expression" in this paragraph (from buildifier) should be replaced with node.

Reply:
Done